Add show password eye on each password input to show/hide passwords

qrainaud opened the pull request:
Added an eye icon on each password input to hide/show passwords.
Added `js/show-password.js` javascript file to be included in `templates/change.tpl` template.
Added `.password-toggle-icon` and `.password-toggle-icon i` in `css/self-service-password.css`.

qrainaud committed:
Add show password eye on each password input to show-hide passwords

qrainaud committed:
Fix of password input right borders after addition of eyes

coudot commented:
We should add a configuration parameter to enable this feature. It should be off by default.

qrainaud committed:
add `$show_pwd_eyes` config setting (false by default) to enable/disable the ability to display password fields with eyes
update `index.php` to pass the setting value to smarty
update `change.tpl` to display eyes and include the associated js file when the setting is enabled

abpai94 commented:
@qrainaud Thank you for the work you did on this issue, I have made some minor changes in my pull request #1125 which builds on your work.
